### PR TITLE
Add members on about page

### PR DESCRIPTION
**Pull request by ahastudio**
Write CEO's information and members list on the `About us` page.

Screenshot

### CODE REVIEW

**Comment by moonkii** [comment]
fighting

File: src/data/members.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+export const ceo = {
+  category: 'ceo',
+  name: 'Sangnin Kwon',
```
Comment (ahastudio) [inline src/data/members.js]: 😱

Comment (ahastudio) [reply src/data/members.js]: https://github.com/microprotect/microprotect.com/pull/87

Comment (canary4651) [reply src/data/members.js]: 제가 급하게 치다가 오타를 냈나봐요....ㅠㅜ

**Pull request merged**